iced_layershell/src/actions.rs

```diff
@@ -94,7 +113,6 @@ impl LayershellCustomActionsWithId {
 }
 
 // first one means
-#[derive(Debug, Clone, Copy)]
```

Comment:
There is no need to delete this derive, I think just deleting the copy is enough

Reply:
i forgot to add it back